.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Do not add this file to the repo, as it contains a setting that can and will change between machines.